cms/templates/simple_html.html

```diff
@@ -0,0 +1,18 @@
+{% extends "catalogue/base.html" %}
+{% block title %} {{ page.title }} - {{ block.super }}{% endblock %}
+{% block body %}
+  <section class="section has-background-primary">
+    <div class="container is-max-widescreen">
+      <h1 class="title has-text-white has-text-centered">
+        {{ page.title }}
+      </h1>
+    </div>
+  </section>
+  <section class="section">
+    <div class="container is-max-widescreen">
+      <div class="content">
+        {{ page.content|safe }}
+      </div>
```

Comment:
`advanced_html.html` should be the same as this one without the last <div>. maybe rename the former to `base.html` and extend it here.